src/main/java/net/snowflake/ingest/utils/Sha256Hash.java

```diff
@@ -0,0 +1,54 @@
+package net.snowflake.ingest.utils;
+
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+import java.util.Base64;
+
+public class Sha256Hash
```

Comment:
Rename to `Cryptor`, so that later all cryptography related functions can all be added into this class. 

Also since this is a util function, it is better to make all method static, so that you don't need to instantiate the class.
When calling the static method, you can do something like
```
Cryptor.sha256HashBase64(publicKeyEncoded);
```

You can take a look at GS code, `com.snowflake.security.Cryptor.java`, (I actually copied sha256Hash function from that class)